File: beginner/pagina14/3302/3302.cpp
#include <cstdio>

using namespace std;

int main() {
	int casos;
	scanf("%d", &casos);

	for(int i = 0; i < casos; i++)
	{
		int curr;
		scanf("%d", &curr);

		printf("resposta %d: %d\n", (i + 1), curr);
	}
	
	return 0;
}

File: beginner/pagina14/3302/3302.rs
use std::io;

fn main() {

    let mut input = String::new();

    io::stdin().read_line(&mut input).unwrap();

    let n: usize = input.trim().parse::<usize>().unwrap_or_default();

    for i in 1..n = 1 {
        input.clear();
        io::stdin().read_line(&mut input).unwrap();
        let curr: &str = input.trim();

        println!("resposta {}: {}", i, curr);
    }
}